app/src/main/res/layout/organiser_fragment_profile.xml

```diff
@@ -43,203 +44,339 @@
+                android:layout_height="wrap_content"
+                android:layout_marginTop="12dp"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:text="Facility Name:"
+                android:textColor="@color/dark_purple"
+                android:textSize="20sp"
+                android:inputType="textPostalAddress"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.325"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_Profile_facilityUploadImage" />
+
+            <EditText
+                android:id="@+id/O_Profile_facilityNameText"
+                android:layout_width="wrap_content"
+                android:layout_height="48dp"
+                android:ems="13"
+                android:focusable="false"
+                android:focusableInTouchMode="false"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:hint="Johnson Recreation Centre"
+                android:textColor="@color/dark_purple"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.503"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_Profile_facilityName" />
+
+            <TextView
+                android:id="@+id/O_Profile_facilityLocation"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="12dp"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:text="Facility Location:"
+                android:textColor="@color/dark_purple"
+                android:textSize="20sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.375"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_Profile_facilityNameText" />
+
+            <EditText
+                android:id="@+id/O_Profile_facilityLocationText"
+                android:layout_width="wrap_content"
+                android:layout_height="48dp"
+                android:ems="13"
+                android:focusable="false"
+                android:focusableInTouchMode="false"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:hint="10323 112 Ave Edmonton"
+                android:textColor="@color/dark_purple"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.503"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_Profile_facilityLocation" />
+
+
+            <Button
+                android:id="@+id/O_P_update"
```

Comment:
This id changes back to the old convention

Reply:
This is not a new id